modules/bcftools/view/meta.yml

```diff
@@ -0,0 +1,42 @@
+name: bcftools_view
+description: View, subset and filter VCF or BCF files by position and filtering expression. Convert between VCF and BCF
+keywords:
+    - variant calling
+    - view
+    - bcftools
+    - VCF
+
+tools:
+    - view:
+        description: |
+          View, subset and filter VCF or BCF files by position and filtering expression. Convert between VCF and BCF
+        homepage: http://samtools.github.io/bcftools/bcftools.html
+        documentation: http://www.htslib.org/doc/bcftools.html
+        doi: 10.1093/bioinformatics/btp352
+input:
+    - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. [ id:'test', single_end:false ]
+    - vcf:
+        type: file
+        description: |
+          The vcf file to be inspected.
+          e.g. 'file1.vcf'
```

Comment:
Missing the optional input files

Reply:
I've addressed these via https://github.com/nf-core/modules/pull/669/commits/77769e3b93c489aca19c403a0555eb0a2584c9bd